### PR TITLE
chore: removing v1 code from the tests

### PR DESCRIPTION
**Pull request by nicolasburtey**

### CODE REVIEW

File: test/integration/services/ledger/transaction.spec.ts
Comment (dolcalmi) [inline test/integration/services/ledger/transaction.spec.ts:108]: please use toSats (applies to other changes in this file)

**Review by dolcalmi — CHANGES_REQUESTED** [review]
looks good, just a couple of comments. @vindard please check before merge (although most of the changes are in the tests we need to be careful, specially with the projections)

File: src/graphql/admin/root/query/transaction-by-id.ts
Comment (nicolasburtey) [reply src/graphql/admin/root/query/transaction-by-id.ts:16]: good catch. I didn't meant to include it, but was pulled over from a rebase. removed.

**Review by vindard — COMMENTED** [review]
Left some comments about variable naming in the tests to be more specific. I'm ok to approve otherwise once [this change](https://github.com/GaloyMoney/galoy/pull/881/files#r780127012) is made

File: test/integration/02-user-wallet/02-is-user-active.spec.ts
Comment (nicolasburtey) [reply test/integration/02-user-wallet/02-is-user-active.spec.ts:11]: done

File: test/integration/02-user-wallet/02-receive-lightning.spec.ts
Comment (nicolasburtey) [reply test/integration/02-user-wallet/02-receive-lightning.spec.ts]: done

File: test/integration/02-user-wallet/02-index.spec.ts
Comment (nicolasburtey) [reply test/integration/02-user-wallet/02-index.spec.ts]: I suggest to do that in another PR

File: test/integration/02-user-wallet/02-index.spec.ts
Comment (nicolasburtey) [reply test/integration/02-user-wallet/02-index.spec.ts]: I suggest to do that in another PR

File: test/integration/02-user-wallet/02-index.spec.ts
Comment (nicolasburtey) [reply test/integration/02-user-wallet/02-index.spec.ts]: done

**Comment by nicolasburtey** [comment]
> Left some comments about variable naming in the tests to be more specific. I'm ok to approve otherwise once [this change](https://github.com/GaloyMoney/galoy/pull/881/files#r780127012) is made

the link for this change doesn't work

**Review by vindard — APPROVED** [review]

**Comment by vindard** [comment]
> @vindard please check before merge (although most of the changes are in the tests we need to be careful, specially with the projections)

@dolcalmi yup checked the projection against the translate function, and anything else in non-test code. Looked ok so far.

**Pull request merged**
